packages/cli-e2e/utils/terminal/terminal.ts

```diff
@@ -35,11 +36,22 @@ export class Terminal {
     const fileLogger = new FileLogger(
       debugName ?? `${command}-${args?.join('-')}`.replace(/[^\w\d]/g, '-')
     );
-    this.childProcess.stdout.pipe(fileLogger.stdout, {end: true});
-    this.childProcess.stderr.pipe(fileLogger.stdout, {end: true});
+
+    this.logIntoFile(fileLogger);
     this.orchestrator = new Orchestrator(this.childProcess);
   }
 
+  private logIntoFile(fileLogger: FileLogger) {
+    const appendTimestamp = (stream: WriteStream) => () =>
+      stream.write(fileLogger.getTimestamp());
+
+    this.childProcess.stdout.on('data', appendTimestamp(fileLogger.stdout));
+    this.childProcess.stderr.on('data', appendTimestamp(fileLogger.stderr));
```

Comment:
I think it'd be better to do something akin to:
chilldProcess.stdout pipe into a 'timestamper' and then the timestamper pipe into the file.